# SOME DESCRIPTIVE TITLE.
# Copyright (C) YEAR THE PACKAGE'S COPYRIGHT HOLDER
# This file is distributed under the same license as the PACKAGE package.
# FIRST AUTHOR <EMAIL@ADDRESS>, YEAR.
#
msgid ""
msgstr ""
"Project-Id-Version: PACKAGE VERSION\n"
"Report-Msgid-Bugs-To: \n"
"POT-Creation-Date: 2025-01-20 12:14-0300\n"
"PO-Revision-Date: YEAR-MO-DA HO:MI+ZONE\n"
"Last-Translator: FULL NAME <EMAIL@ADDRESS>\n"
"Language-Team: LANGUAGE <LL@li.org>\n"
"Language: \n"
"MIME-Version: 1.0\n"
"Content-Type: text/plain; charset=UTF-8\n"
"Content-Transfer-Encoding: 8bit\n"
"Plural-Forms: nplurals=2; plural=(n > 1);\n"

#: whatsapp_app/api/v1/calificacion.py:209
msgid "respuestaFormularioGestion es obligatorio"
msgstr "respuestaFormularioGestion e obrigatorio"

#: whatsapp_app/api/v1/calificacion.py:222
#: whatsapp_app/api/v1/calificacion.py:249
msgid "error en los campos del formulario"
msgstr "erro nos campos do formulario"

#: whatsapp_app/api/v1/calificacion.py:264
msgid "Se obtuvo la calificacion de forma exitosa"
msgstr "Qualificação obtida com sucesso"

#: whatsapp_app/api/v1/calificacion.py:270
#: whatsapp_app/api/v1/calificacion.py:429
msgid "Calificacion no encontrada"
msgstr "Qualificação não encontrada"

#: whatsapp_app/api/v1/calificacion.py:276
msgid "Error al obtener las calificacio"
msgstr "Erro obtendo qualificação"

#: whatsapp_app/api/v1/calificacion.py:309
#: whatsapp_app/api/v1/calificacion.py:383
msgid "Error en los datos del formulario"
msgstr "Erro nos dados do formulário"

#: whatsapp_app/api/v1/calificacion.py:316
#: whatsapp_app/api/v1/calificacion.py:332
msgid "Se creo la calificacion de forma exitosa"
msgstr "Qualificação criada com sucesso"

#: whatsapp_app/api/v1/calificacion.py:338
#: whatsapp_app/api/v1/calificacion.py:401
#: whatsapp_app/api/v1/configuracion_whatsapp_campana.py:129
#: whatsapp_app/api/v1/configuracion_whatsapp_campana.py:174
#: whatsapp_app/api/v1/contacto.py:240 whatsapp_app/api/v1/contacto.py:267
#: whatsapp_app/api/v1/contacto.py:296
#: whatsapp_app/api/v1/grupo_plantilla_whatsapp.py:113
#: whatsapp_app/api/v1/grupo_plantilla_whatsapp.py:157
#: whatsapp_app/api/v1/linea.py:82 whatsapp_app/api/v1/linea.py:109
#: whatsapp_app/api/v1/linea.py:114 whatsapp_app/api/v1/linea.py:171
#: whatsapp_app/api/v1/linea.py:201 whatsapp_app/api/v1/linea.py:206
#: whatsapp_app/api/v1/plantilla_mensaje.py:144
#: whatsapp_app/api/v1/plantilla_mensaje.py:207
#: whatsapp_app/api/v1/proveedor.py:184 whatsapp_app/api/v1/proveedor.py:247
msgid "Error en los datos"
msgstr "Erro nos dados"

#: whatsapp_app/api/v1/calificacion.py:395
msgid "Se actualizo la calificacion de forma exitosa"
msgstr "Qualificação atualizada com sucesso"

#: whatsapp_app/api/v1/calificacion.py:406
msgid "CalificacionCliente no encontrada"
msgstr "CalificacionCliente não encontrada"

#: whatsapp_app/api/v1/calificacion.py:412
msgid "Error al actualizar la CalificacionCliente"
msgstr "Erro al atualizar CalificacionCliente"

#: whatsapp_app/api/v1/calificacion.py:435
msgid "Error al obtener el historial de calificacion"
msgstr "Erro obtendo o histórico da Qualificação"

#: whatsapp_app/api/v1/calificacion.py:448
msgid "Se obtuvieron las opciones de calificacion de forma exitosa"
msgstr "Opções de qualificação obtidas com sucesso"

#: whatsapp_app/api/v1/calificacion.py:455
msgid "Error al obtener opciones de calificacion"
msgstr "Erro obtendo opções de qualificação"

#: whatsapp_app/api/v1/campana.py:69
msgid "Se obtuvieron las campanas de forma exitosa"
msgstr "Campanhas obtidas com sucesso"

#: whatsapp_app/api/v1/campana.py:76
msgid "Error al obtener campanas"
msgstr "Erro ao obter campanhas"

#: whatsapp_app/api/v1/configuracion_whatsapp_campana.py:102
msgid "Se obtuvieron las configuraciones de whatsapp de forma exitosa"
msgstr "Configurações de Whatsapp obtidas com sucesso"

#: whatsapp_app/api/v1/configuracion_whatsapp_campana.py:109
msgid "Error al obtener las configuraciones de whatsapp"
msgstr "Erro obtendo configurações de whatsapp"

#: whatsapp_app/api/v1/configuracion_whatsapp_campana.py:123
msgid "Se creo la configuraciones de whatsapp de forma exitosa"
msgstr "Configuração d whatsapp criada com sucesso"

#: whatsapp_app/api/v1/configuracion_whatsapp_campana.py:134
msgid "Error al crear la configuraciones de whatsapp"
msgstr "Erro criando configurações de whatsapp"

#: whatsapp_app/api/v1/configuracion_whatsapp_campana.py:146
msgid "Se obtuvo la configuracion de whatsapp de forma exitosa"
msgstr "Configuração de whatsapp obtida com sucesso"

#: whatsapp_app/api/v1/configuracion_whatsapp_campana.py:150
msgid "Configuracion de whatsapp no encontrada"
msgstr "Confguração de whatssap não encontrada"

#: whatsapp_app/api/v1/configuracion_whatsapp_campana.py:155
msgid "Error al obtener la configuracion de whatsapp"
msgstr "Erro obtendo configuração de whatsapp"

#: whatsapp_app/api/v1/configuracion_whatsapp_campana.py:169
msgid "Se actualizó la configuracion de whatsapp de forma exitosa"
msgstr "Configuração de whatsapp atualizada com sucesso"

#: whatsapp_app/api/v1/configuracion_whatsapp_campana.py:178
msgid "Configuración de whatsapp no encontrada"
msgstr "Configuração de whatsapp não encontrada"

#: whatsapp_app/api/v1/configuracion_whatsapp_campana.py:183
msgid "Error al actualizar la configuaración de whatsapp"
msgstr "Erro atualizando Configuração de whatsapp"

#: whatsapp_app/api/v1/configuracion_whatsapp_campana.py:194
msgid "Se elimino la configuración de whatsapp de forma exitosa"
msgstr "Configuração de whatsapp excluída com sucesso"

#: whatsapp_app/api/v1/configuracion_whatsapp_campana.py:198
msgid "Configuración de whatsapp no encontrado"
msgstr "Configuração de whatsapp não encontrada"

#: whatsapp_app/api/v1/configuracion_whatsapp_campana.py:203
msgid "Error al eliminar la Configuración de whatsapp"
msgstr "Erro ao excluir Configuração de whatsapp"

#: whatsapp_app/api/v1/contacto.py:85 whatsapp_app/api/v1/contacto.py:97
#: whatsapp_app/api/v1/contacto.py:150 whatsapp_app/api/v1/contacto.py:162
msgid "Debe ser en formato \"999999999\" y solo numérico."
msgstr "Deve ter formato \"999999999\" e somente numérico."

#: whatsapp_app/api/v1/contacto.py:88 whatsapp_app/api/v1/contacto.py:100
#: whatsapp_app/api/v1/contacto.py:153 whatsapp_app/api/v1/contacto.py:165
msgid "Solo se permiten de 3-20 dígitos."
msgstr "Somente são permitidos de 3-20 dígitos."

#: whatsapp_app/api/v1/contacto.py:116
msgid "campo requerido"
msgstr "campo solicitado"

#: whatsapp_app/api/v1/contacto.py:121
msgid "Faltan campos requeridos"
msgstr "Faltam campos solicitados"

#: whatsapp_app/api/v1/contacto.py:123 whatsapp_app/api/v1/contacto.py:189
msgid "Error en los campos de contacto"
msgstr "Erro nos campos de contato"

#: whatsapp_app/api/v1/contacto.py:187
msgid "No puede editar campos ocultos o bloqueados"
msgstr "Não pode editar campos ocultos ou bloqueados"

#: whatsapp_app/api/v1/contacto.py:210
msgid "Se obtuvieron las contactos de forma exitosa"
msgstr "Contatos obtidos com sucesso"

#: whatsapp_app/api/v1/contacto.py:235 whatsapp_app/api/v1/contacto.py:262
msgid "Se creo el nuevo contacto de forma exitosa"
msgstr "Novo contato criado com sucesso"

#: whatsapp_app/api/v1/contacto.py:245 whatsapp_app/api/v1/contacto.py:272
msgid "Error al crear el contacto"
msgstr "Erro criando contato"

#: whatsapp_app/api/v1/contacto.py:290
msgid "Se actualizo el nuevo contacto de forma exitosa"
msgstr "Contato atualizado com sucesso"

#: whatsapp_app/api/v1/contacto.py:300
msgid "Contacto no encontrado"
msgstr "Contato não encontrado"

#: whatsapp_app/api/v1/contacto.py:306
msgid "Error al actualizar el contacto"
msgstr "Erro atualizando o vontato"

#: whatsapp_app/api/v1/contacto.py:332
msgid "Error al obtener los campos de contacto"
msgstr "Erro ao obter os campos do contato"

#: whatsapp_app/api/v1/contacto.py:359
msgid "Se obtuvieron los contactos de forma exitosa"
msgstr "Contatos obtidos com sucesso"

#: whatsapp_app/api/v1/contacto.py:366
msgid "Error al obtener contactos"
msgstr "Erro ao obter contatos"

#: whatsapp_app/api/v1/conversacion.py:211
msgid "Se obtuvieron las conversaciones de forma exitosa"
msgstr "Conversas obtidas com sucesso"

#: whatsapp_app/api/v1/conversacion.py:220
msgid "Error al obtener las conversaciones"
msgstr "Erro obtendo conversas"

#: whatsapp_app/api/v1/conversacion.py:232
msgid "Se obtuvo la conversacion de forma exitosa"
msgstr "Conversa obtida com sucesso"

#: whatsapp_app/api/v1/conversacion.py:236
msgid "Conversacion no encontrada"
msgstr "Conversa não encontrada"

#: whatsapp_app/api/v1/conversacion.py:242
msgid "Error al obtener la conversacion"
msgstr "Erro obtendo conversa"

#: whatsapp_app/api/v1/conversacion.py:285
msgid "Se asignó la conversación de forma exitosa"
msgstr "Conversa asignada com sucesso"

#: whatsapp_app/api/v1/conversacion.py:290
#: whatsapp_app/api/v1/conversacion.py:376
#: whatsapp_app/api/v1/conversacion.py:433
#: whatsapp_app/api/v1/conversacion.py:509
#: whatsapp_app/api/v1/conversacion.py:566
msgid "Esta conversación ya está siendo atendida por otro agente"
msgstr "Essa conversa esta sendo atendida por otro agente"

#: whatsapp_app/api/v1/conversacion.py:295
#: whatsapp_app/api/v1/conversacion.py:331
msgid "No se puede asignar una conversación que no existe"
msgstr "Não pode asignar uma conversa que não existe"

#: whatsapp_app/api/v1/conversacion.py:301
msgid "Error al asignar la conversación"
msgstr "Erro ao asignar conversa"

#: whatsapp_app/api/v1/conversacion.py:313
msgid "El contacto no pertenece a la base de datos de la campaña"
msgstr "O contato não pertence ao banco de dados da campanha"

#: whatsapp_app/api/v1/conversacion.py:319
msgid "El contacto ya tiene una conversación activa"
msgstr "O contato ja possui uma conversa ativa"

#: whatsapp_app/api/v1/conversacion.py:326
msgid "Se asigno el contacto a la conversacion de forma satisfactoria"
msgstr "O contato foi atribuído à conversa com sucesso"

#: whatsapp_app/api/v1/conversacion.py:336
msgid "No existe el contacto que se quiere asignar"
msgstr "Não existe o contato indicado para atribuir"

#: whatsapp_app/api/v1/conversacion.py:343
msgid "Error al asignar el contacto a la conversación"
msgstr "Erro ao atribuir o contato à conversa"

#: whatsapp_app/api/v1/conversacion.py:395
#: whatsapp_app/api/v1/conversacion.py:471
msgid "Este mensaje no se pudo enviar"
msgstr "Essa mensagem não pudo ser enviada"

#: whatsapp_app/api/v1/conversacion.py:404
#: whatsapp_app/api/v1/conversacion.py:480
#: whatsapp_app/api/v1/conversacion.py:534
#: whatsapp_app/api/v1/conversacion.py:610
msgid "La conversacion esta inactiva hasta que el cliente responda"
msgstr "A conversa está inativa até o cliente responder"

#: whatsapp_app/api/v1/conversacion.py:407
#: whatsapp_app/api/v1/conversacion.py:483
#: whatsapp_app/api/v1/conversacion.py:539
#: whatsapp_app/api/v1/conversacion.py:613
msgid "La conversacion ha expirado. Inicie una nueva conversacion."
msgstr "A conversa expirou. Comece uma nova conversa."

#: whatsapp_app/api/v1/conversacion.py:411
#: whatsapp_app/api/v1/conversacion.py:487
msgid "Conversacion es erronea"
msgstr "Conversa errada"

#: whatsapp_app/api/v1/conversacion.py:418
#: whatsapp_app/api/v1/conversacion.py:494
#: whatsapp_app/api/v1/conversacion.py:549
#: whatsapp_app/api/v1/conversacion.py:622
#: whatsapp_app/api/v1/conversacion.py:694
#: whatsapp_app/api/v1/conversacion.py:800
msgid "Error al enviar el mensaje"
msgstr "Erro enviando mensagem"

#: whatsapp_app/api/v1/conversacion.py:529
#: whatsapp_app/api/v1/conversacion.py:604
msgid "Se envió el mensaje de forma exitosa"
msgstr "A mensagem foi enviada com sucesso"

#: whatsapp_app/api/v1/conversacion.py:543
#: whatsapp_app/api/v1/conversacion.py:617
#: whatsapp_app/api/v1/conversacion.py:640
msgid "Conversacion erronea"
msgstr "Conversa errada"

#: whatsapp_app/api/v1/conversacion.py:645
msgid "Ya existe una conversacion iniciada con el cliente"
msgstr "Ja existe uma conversa iniciada com o cliente"

#: whatsapp_app/api/v1/conversacion.py:686
msgid "Se envio el mensaje de forma exitosa"
msgstr "A mensagem foi enviada com sucesso"

#: whatsapp_app/api/v1/conversacion.py:709
#, python-brace-format
msgid "Campos esperados: {0}"
msgstr "Campos esperados: {0}"

#: whatsapp_app/api/v1/conversacion.py:716
#: whatsapp_app/api/v1/conversacion.py:721
#: whatsapp_app/api/v1/conversacion.py:726
msgid "Campaña inválida"
msgstr "Campanha errada"

#: whatsapp_app/api/v1/conversacion.py:732
msgid "Template inválido"
msgstr "Template inválido"

#: whatsapp_app/api/v1/conversacion.py:743
msgid "Contacto inválido"
msgstr "Contato inválido"

#: whatsapp_app/api/v1/conversacion.py:790
msgid "Conversacion creada correctamente"
msgstr "Conversa criada com sucesso"

#: whatsapp_app/api/v1/conversacion.py:794
msgid "Ya existe una conversacion iniciada"
msgstr "Ja existe uma conversa iniciada"

#: whatsapp_app/api/v1/destino.py:51
msgid "Se obtuvieron los destinos de forma exitosa"
msgstr "Destinos obtidos com sucesso"

#: whatsapp_app/api/v1/destino.py:57
#: whatsapp_app/api/v1/grupo_plantilla_whatsapp.py:93
msgid "Error al obtener los destinos"
msgstr "Erro ao obter os destinos"

#: whatsapp_app/api/v1/grupo_plantilla_whatsapp.py:87
msgid "Se obtuvieron los grupo plantilla mensaje de forma exitosa"
msgstr "Grupos de modelos de mensagem obtidos com sucesso"

#: whatsapp_app/api/v1/grupo_plantilla_whatsapp.py:107
msgid "Se creo grupo plantilla mensaje de forma exitosa"
msgstr "Grupo de modelos de mensagem criado com sucesso"

#: whatsapp_app/api/v1/grupo_plantilla_whatsapp.py:117
msgid "Error al crear el grupo plantilla mensaje"
msgstr "Erro criando grupo de modelos de mensagem"

#: whatsapp_app/api/v1/grupo_plantilla_whatsapp.py:129
msgid "Se obtuvo grupo plantilla mensaje de forma exitosa"
msgstr "Grupo de modelos de mensagem obtido com sucesso"

#: whatsapp_app/api/v1/grupo_plantilla_whatsapp.py:133
#: whatsapp_app/api/v1/grupo_plantilla_whatsapp.py:161
msgid "Grupo plantilla mensaje no encontrada"
msgstr "Grupo de modelos de mensagem não encontrada"

#: whatsapp_app/api/v1/grupo_plantilla_whatsapp.py:138
msgid "Error al obtener grupo plantilla mensaje"
msgstr "Erro obtendo grupo de modelos de mensagem"

#: whatsapp_app/api/v1/grupo_plantilla_whatsapp.py:152
msgid "Se actualizó grupo plantilla mensaje de forma exitosa"
msgstr "Grupo de modelos de mensagem atualizado com sucesso"

#: whatsapp_app/api/v1/grupo_plantilla_whatsapp.py:166
msgid "Error al actualizar grupo plantilla mensaje"
msgstr "Erro al atualizar grupo de modelos de mensagem"

#: whatsapp_app/api/v1/grupo_plantilla_whatsapp.py:178
msgid "Se elimino grupo plantilla mensaje de forma exitosa"
msgstr "Grupo de modelos de mensagem excluído com sucesso"

#: whatsapp_app/api/v1/grupo_plantilla_whatsapp.py:183
msgid "Grupo plantilla está siendo usado por alguna campaña activa"
msgstr ""
"Grupo de modelos de mensagem esta sendo utilizado por uma campanha ativa"

#: whatsapp_app/api/v1/grupo_plantilla_whatsapp.py:187
msgid "Grupo plantilla mensaje no encontrado"
msgstr "Grupo de modelos de mensagem não encontrado"

#: whatsapp_app/api/v1/grupo_plantilla_whatsapp.py:192
msgid "Error al eliminar Grupo plantilla mensaje"
msgstr "Erro ao excluir grupo de modelos de mensagem"

#: whatsapp_app/api/v1/linea.py:68
msgid "Se obtuvieron las líneas de forma exitosa"
msgstr "Linhas obtidas com sucesso"

#: whatsapp_app/api/v1/linea.py:74
msgid "Error al obtener las líneas"
msgstr "Erro ao obter linhas"

#: whatsapp_app/api/v1/linea.py:83 whatsapp_app/api/v1/linea.py:172
msgid "Este campo es requerido."
msgstr "Campo obrigatório"

#: whatsapp_app/api/v1/linea.py:104 whatsapp_app/api/v1/linea.py:196
msgid "Se creo la línea de forma exitosa"
msgstr "Linha criada com sucesso"

#: whatsapp_app/api/v1/linea.py:120 whatsapp_app/api/v1/linea.py:212
msgid "Error al crear la línea"
msgstr "Erro ao criar a linha"

#: whatsapp_app/api/v1/linea.py:151
msgid "Se obtuvo la línea de forma exitosa"
msgstr "Linha obtida com sucesso"

#: whatsapp_app/api/v1/linea.py:155
msgid "Línea no encontrada"
msgstr "Linha não encontrada"

#: whatsapp_app/api/v1/linea.py:161
msgid "Error al obtener la línea"
msgstr "Erro ao obter a linha"

#: whatsapp_app/api/v1/linea.py:227
msgid "Se elimino la línea de forma exitosa"
msgstr "Linha excluída com sucesso"

#: whatsapp_app/api/v1/linea.py:232
msgid "Esta línea está siendo usada por alguna campaña activa."
msgstr "Essa linha esta sendo utilizada por alguma campanha ativa."

#: whatsapp_app/api/v1/linea.py:236
msgid "Línea no encontrado"
msgstr "Linha não encontrada"

#: whatsapp_app/api/v1/linea.py:241
msgid "Error al eliminar la línea"
msgstr "Erro ao eliminar a linha"

#: whatsapp_app/api/v1/linea_serializers.py:85
#: whatsapp_app/api/v1/linea_serializers.py:93
#: whatsapp_app/api/v1/linea_serializers.py:446
#: whatsapp_app/api/v1/linea_serializers.py:455
#: whatsapp_app/api/v1/proveedor.py:72 whatsapp_app/api/v1/proveedor.py:77
#: whatsapp_app/api/v1/proveedor.py:115 whatsapp_app/api/v1/proveedor.py:120
msgid "Configuración incorrecta para el tipo de proveedor"
msgstr "Configuração errada para o tipo de fornecedor"

#: whatsapp_app/api/v1/linea_serializers.py:88
#: whatsapp_app/api/v1/linea_serializers.py:450
msgid "Ya existe una Linea con ese App ID"
msgstr "Ja existe uma linha com esse App ID"

#: whatsapp_app/api/v1/linea_serializers.py:115
msgid "Campana"
msgstr "Campanha"

#: whatsapp_app/api/v1/linea_serializers.py:116
msgid "Menu Interactivo"
msgstr "Menú interativo"

#: whatsapp_app/api/v1/linea_serializers.py:132
msgid "Valor incorrecto. Debe ser un id"
msgstr "Valor errado. Deve ser um id"

#: whatsapp_app/api/v1/linea_serializers.py:138
msgid "Valor incorrecto. La campaña no tiene whatsapp habilitado"
msgstr "Valor errado. A campanha não tem whatsapp habilitado"

#: whatsapp_app/api/v1/linea_serializers.py:143
msgid "No existe destino con ese id de Campaña"
msgstr "Não existe destino com esse id de campanha"

#: whatsapp_app/api/v1/linea_serializers.py:150
msgid "Valor incorrecto."
msgstr "Valor errado."

#: whatsapp_app/api/v1/linea_serializers.py:325
msgid "El valor de las opciones no puede repetirse"
msgstr "O valor das opções não pode se repetir"

#: whatsapp_app/api/v1/linea_serializers.py:328
msgid "No pueden definirse más de 10 opciones"
msgstr "Não podem se definir mais de 10 opções"

#: whatsapp_app/api/v1/linea_serializers.py:431
msgid "Debe indicar una configuración de proveedor."
msgstr "Deve indicar uma configuração de fornecedor"

#: whatsapp_app/api/v1/linea_serializers.py:437
msgid "Debe indicar un proveedor."
msgstr "Deve indicar um fornecedor."

#: whatsapp_app/api/v1/mensaje.py:114
msgid "Se obtuvieron los mensajes de forma exitosa"
msgstr "Mensagems obtidos com sucesso"

#: whatsapp_app/api/v1/mensaje.py:120
msgid "Error al obtener los mensajes"
msgstr "Erro ao obter as mensagems"

#: whatsapp_app/api/v1/plantilla_mensaje.py:58
#: whatsapp_app/api/v1/plantilla_mensaje.py:96
msgid "No soportado por el momento"
msgstr "Não suportado por enquanto"

#: whatsapp_app/api/v1/plantilla_mensaje.py:65
#: whatsapp_app/api/v1/plantilla_mensaje.py:103
msgid "Configuración incorrecta para el tipo de mensaje"
msgstr "Configuração errada para o tipo de mensagem"

#: whatsapp_app/api/v1/plantilla_mensaje.py:118
#: whatsapp_app/api/v1/template_whatsapp.py:77
msgid "Se obtuvieron las plantillas de forma exitosa"
msgstr "Modelos obtidos com sucesso"

#: whatsapp_app/api/v1/plantilla_mensaje.py:124
#: whatsapp_app/api/v1/template_whatsapp.py:84
msgid "Error al obtener las plantillas"
msgstr "Erro ao obter os modelos"

#: whatsapp_app/api/v1/plantilla_mensaje.py:138
msgid "Se creo la platilla de forma exitosa"
msgstr "Modelo criado com sucesso"

#: whatsapp_app/api/v1/plantilla_mensaje.py:148
msgid "Error al crear la plantilla"
msgstr "Erro ao criar o modelo"

#: whatsapp_app/api/v1/plantilla_mensaje.py:179
msgid "Se obtuvo la plantilla de forma exitosa"
msgstr "O modelo foi obtido com sucesso"

#: whatsapp_app/api/v1/plantilla_mensaje.py:183
#: whatsapp_app/api/v1/plantilla_mensaje.py:211
msgid "Plantilla no encontrada"
msgstr "Modelo não encontrado"

#: whatsapp_app/api/v1/plantilla_mensaje.py:188
msgid "Error al obtener la plantilla"
msgstr "Erro ao obter o modelo"

#: whatsapp_app/api/v1/plantilla_mensaje.py:202
msgid "Se actualizó la plantilla de forma exitosa"
msgstr "Modelo atualizado com sucesso"

#: whatsapp_app/api/v1/plantilla_mensaje.py:216
msgid "Error al actualizar la plantilla"
msgstr "Erro ao atualizar o modelo"

#: whatsapp_app/api/v1/plantilla_mensaje.py:227
msgid "Se elimino la plantilla de forma exitosa"
msgstr "Modelo excluído com sucesso"

#: whatsapp_app/api/v1/plantilla_mensaje.py:231
msgid "Plantilla no encontrado"
msgstr "Modelo não encontrado"

#: whatsapp_app/api/v1/plantilla_mensaje.py:236
msgid "Error al eliminar la Plantilla"
msgstr "Erro ao eliminar o modelo"

#: whatsapp_app/api/v1/proveedor.py:158
msgid "Se obtuvieron las configuraciones de proveedor de forma exitosa"
msgstr "Configurações de fornecedor obtidas com sucesso"

#: whatsapp_app/api/v1/proveedor.py:164
msgid "Error al obtener las configuraciones de proveedor"
msgstr "Erro ao obter as configurações de fornecedor"

#: whatsapp_app/api/v1/proveedor.py:178
msgid "Se creo la configuaración de proveedor de forma exitosa"
msgstr "Configuração de fornecedor criada com sucesso"

#: whatsapp_app/api/v1/proveedor.py:188
msgid "Error al crear la configuaración de proveedor"
msgstr "Erro ao criar Configuração de fornecedor"

#: whatsapp_app/api/v1/proveedor.py:219
msgid "Se obtuvo la configuración del proveedor de forma exitosa"
msgstr "Configuração de fornecedor obtida com sucesso"

#: whatsapp_app/api/v1/proveedor.py:223 whatsapp_app/api/v1/proveedor.py:251
msgid "Configuración del proveedor no encontrada"
msgstr "Configuração de fornecedor não encontrada"

#: whatsapp_app/api/v1/proveedor.py:228
msgid "Error al obtener la configuración del proveedor"
msgstr "Erro obtendo Configuração de fornecedor"

#: whatsapp_app/api/v1/proveedor.py:242
msgid "Se actualizó la configuración del proveedor de forma exitosa"
msgstr "Configuração de fornecedor atualizada com sucesso"

#: whatsapp_app/api/v1/proveedor.py:256
msgid "Error al actualizar la configuaración del proveedor"
msgstr "Erro atualizando Configuração de fornecedor"

#: whatsapp_app/api/v1/proveedor.py:268
msgid "Se elimino la Configuración del proveedor de forma exitosa"
msgstr "Configuração de fornecedor excluída com sucesso"

#: whatsapp_app/api/v1/proveedor.py:273
msgid "Proveedor con líneas activas."
msgstr "Fornecedor com linhas ativas"

#: whatsapp_app/api/v1/proveedor.py:277
msgid "Configuración del proveedor no encontrado"
msgstr "Configuração de fornecedor não encontrada"

#: whatsapp_app/api/v1/proveedor.py:282
msgid "Error al eliminar la Configuración del proveedor"
msgstr "Erro ao eliminar Configuração de fornecedor"

#: whatsapp_app/api/v1/reporte.py:140
msgid "Error al obtener el reporte"
msgstr "Erro obtendo o relatório"

#: whatsapp_app/api/v1/reporte.py:145
msgid "Reporte obtenido exitosamente"
msgstr "Relatório obtido com sucesso"

#: whatsapp_app/api/v1/template.py:57
msgid "Se obtuvieron los templates de forma exitosa"
msgstr "Modelos obtidos com sucesso"

#: whatsapp_app/api/v1/template_whatsapp.py:95
msgid "Se obtuvo el template de forma exitosa"
msgstr "Modelo obtido com sucesso"

#: whatsapp_app/api/v1/template_whatsapp.py:99
msgid "Template no encontrada"
msgstr "Modelo não encontrado"

#: whatsapp_app/api/v1/template_whatsapp.py:104
msgid "Error al obtener el Template"
msgstr "Erro obtendo o modelo"

#: whatsapp_app/api/v1/template_whatsapp.py:136
msgid "Se obtuvieron los templates de whatsapp de forma exitosa"
msgstr "Modelos de whatsapp obtidos com sucesso"

#: whatsapp_app/api/v1/template_whatsapp.py:153
msgid "Se cambio el estado del template de forma exitosa"
msgstr "O estado do modelo foi modificado com sucesso"

#: whatsapp_app/api/v1/template_whatsapp.py:159
msgid "No tiene permiso para esta accion"
msgstr "Não tem permissão para essa ação"

#: whatsapp_app/api/v1/transfer.py:67
msgid "Error al obtener los agentes"
msgstr "Erro obtendo agentes"

#: whatsapp_app/api/v1/transfer.py:89 whatsapp_app/api/v1/transfer.py:95
msgid "Error al tranferir conversacion"
msgstr "Erro ao transferir conversa"

#: whatsapp_app/apps.py:115
msgid "whatsapp_providers_configuration"
msgstr "whatsapp_providers_configuration"

#: whatsapp_app/apps.py:118
msgid "whatsapp_lines_configuration"
msgstr "whatsapp_lines_configuration"

#: whatsapp_app/apps.py:121
msgid "message_templates_configuration"
msgstr "message_templates_configuration"

#: whatsapp_app/apps.py:124
msgid "whatsapp_message_template_groups"
msgstr "whatsapp_message_template_groups"

#: whatsapp_app/apps.py:127
msgid "proveedor-list"
msgstr "proveedor-list"

#: whatsapp_app/apps.py:130
msgid "proveedor-detail"
msgstr "proveedor-detail"

#: whatsapp_app/apps.py:133
msgid "linea-list"
msgstr "linea-list"

#: whatsapp_app/apps.py:136
msgid "linea-detail"
msgstr "linea-detail"

#: whatsapp_app/apps.py:139
msgid "destino-list"
msgstr "destino-list"

#: whatsapp_app/apps.py:142
msgid "plantilla-mensaje-list"
msgstr "plantilla-mensaje-list"

#: whatsapp_app/apps.py:145
msgid "plantilla-mensaje-detail"
msgstr "plantilla-mensaje-detail"

#: whatsapp_app/apps.py:148
msgid "template-whatsapp-list"
msgstr "template-whatsapp-list"

#: whatsapp_app/apps.py:151
msgid "template-whatsapp-sincronizar-templates"
msgstr "template-whatsapp-sincronizar-templates"

#: whatsapp_app/apps.py:154
msgid "campana-list"
msgstr "campana-list"

#: whatsapp_app/apps.py:157
msgid "grupo-plantilla-whatsapp-listp"
msgstr "grupo-plantilla-whatsapp-listp"

#: whatsapp_app/apps.py:160
msgid "grupo-plantilla-whatsapp-detail"
msgstr "grupo-plantilla-whatsapp-detail"

#: whatsapp_app/apps.py:163
msgid "grupo-template-whatsapp-list"
msgstr "grupo-template-whatsapp-list"

#: whatsapp_app/apps.py:166
msgid "grupo-template-whatsapp-detail"
msgstr "grupo-template-whatsapp-detail"

#: whatsapp_app/apps.py:169
msgid "configuracion-whatsapp-campana-list"
msgstr "configuracion-whatsapp-campana-list"

#: whatsapp_app/apps.py:172
msgid "configuracion-whatsapp-campana-templates"
msgstr "configuracion-whatsapp-campana-templates"

#: whatsapp_app/apps.py:175
msgid "configuracion-whatsapp-campana-detai"
msgstr "configuracion-whatsapp-campana-detai"

#: whatsapp_app/apps.py:178
msgid "conversacion-list"
msgstr "conversacion-list"

#: whatsapp_app/apps.py:181
msgid "conversacion-agent-chats-lists"
msgstr "conversacion-agent-chats-lists"

#: whatsapp_app/apps.py:184
msgid "conversacion-attend-chat"
msgstr "conversacion-attend-chat"

#: whatsapp_app/apps.py:187
msgid "conversacion-messages"
msgstr "conversacion-messages"

#: whatsapp_app/apps.py:190
msgid "conversacion-send-menssage-attachment"
msgstr "conversacion-send-menssage-attachment"

#: whatsapp_app/apps.py:193
msgid "conversacion-send-menssage-text"
msgstr "conversacion-send-menssage-text"

#: whatsapp_app/apps.py:196
msgid "conversacion-send-message-template"
msgstr "conversacion-send-message-template"

#: whatsapp_app/apps.py:199
msgid "conversacion-send-message-whatsapp-template"
msgstr "conversacion-send-message-whatsapp-template"

#: whatsapp_app/apps.py:202
msgid "transfer-to-agent"
msgstr "transfer-to-agent"

#: whatsapp_app/apps.py:205
msgid "transfer-agents"
msgstr "transfer-agents"

#: whatsapp_app/apps.py:208
msgid "conversacion-assign-contact"
msgstr "conversacion-assign-contact"

#: whatsapp_app/apps.py:211
msgid "contacto-search"
msgstr "contacto-search"

#: whatsapp_app/apps.py:214
msgid "contacto-list"
msgstr "contacto-list"

#: whatsapp_app/apps.py:217
msgid "contacto-detail"
msgstr "contacto-detail"

#: whatsapp_app/apps.py:220
msgid "contacto-db-fields"
msgstr "contacto-db-fields"

#: whatsapp_app/apps.py:223
msgid "calificacion-list"
msgstr "calificacion-list"

#: whatsapp_app/apps.py:226
msgid "calificacion-options"
msgstr "calificacion-options"

#: whatsapp_app/apps.py:229
msgid "calificacion-detail"
msgstr "calificacion-detail"

#: whatsapp_app/apps.py:232
msgid "calificacion-history"
msgstr "calificacion-history"

#: whatsapp_app/apps.py:235
msgid "whatsapp_reports"
msgstr "whatsapp_reports"

#: whatsapp_app/models.py:40
msgid "Twilio"
msgstr "Twilio"

#: whatsapp_app/models.py:41
msgid "Meta"
msgstr "Meta"

#: whatsapp_app/models.py:42
msgid "GupShup"
msgstr "GupShup"

#: whatsapp_app/models.py:93
msgid "Texto"
msgstr "Texto"

#: whatsapp_app/models.py:94
msgid "Imagen"
msgstr "Imagem"
